src/routes/give-away/get.js

```diff
@@ -0,0 +1,53 @@
+import Joi from 'joi'
+import RestifyErrors from 'restify-errors'
+
+import models from '../../models'
+
+export default {
+
+  method: 'get',
+
+  // @TODO: How to filter?
+  path: '/giveaways?',
+
+  parameters: Joi.object().keys({
+    active: Joi.boolean(),
+  }),
+
+  handler(request, response) {
+
+    // const populateConditions = request.parameters.include.map((active) => {
+    //   return {
+    //   }
+    // })
+
+    // @TODO: Filter for ones the user has not participated in
+
+    if (request.params.giveawayId) {
```

Comment:
This should be listed in the Joi-validated `parameters` above and read from `request.parameters` instead.

Reply:
Oh okay, I just realized you combined the get & find routes into one file. Please break them out into `get.js` and `find.js`. The `get.js` would then have the `giveawayId` specified in the route params and read from `request.params.giveawayId`.